Headers/species_mod.F90

```diff
@@ -158,6 +164,25 @@ MODULE Species_Mod
      REAL(fp)           :: WD_RainoutEff(3) ! Temperature-dependent scale
                                             !  factors for rainout efficiency
 
+     ! TransportTracers parameters
+     CHARACTER(LEN=80)  :: Snk_Horiz        ! Where to apply sink horizontally?
```

Comment:
What are the options for this? Same question for Src.

Reply:
The options are now specified on RTD under [Transport tracer properties](https://geos-chem.readthedocs.io/en/latest/geos-chem-shared-docs/supplemental-guides/species-guide.html#transport-tracer-properties).